src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class User {
+    private int id;
```

Comment:
is this `id` set in our program?
if no, let's maybe remove it

Reply:
removed...done